### PR TITLE
fix(Dialog): align Dialog.Action to left when used in information variant

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

**Review by langz — APPROVED** [review]
⚡Too fast⚡

File: packages/dnb-eufemia/src/components/dialog/__tests__/DialogAction.test.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
+import React from 'react'
+import Dialog from '../Dialog'
+import Button from '../../button/Button'
+// import Provider from '../../../shared/Provider'
+// import {
+//   mount,
+//   // fakeProps,
+//   toJson,
+//   loadScss,
+//   axeComponent,
```
Comment (langz) [inline packages/dnb-eufemia/src/components/dialog/__tests__/DialogAction.test.tsx:10]: A tad bit many `//` comments 💬

Comment (tujoworker) [reply packages/dnb-eufemia/src/components/dialog/__tests__/DialogAction.test.tsx:10]: ohh, yes! Thank you!

**Pull request merged**